NBA-Stats-Dashboard/src/components/Header.jsx

```diff
@@ -1,7 +1,12 @@
 function Header() {
   return (
     <div className='majorSection header'>
-      <h1>The Bannner will go here. Which is inside the Header Component.</h1>
+      <img
+          src="../src/assets/banner.jpeg"
+          height="200px"
+          width="100%"
```

Comment:
I feel like a 100% width makes the banner image look distorted at certain screen sizes, especially large (desktop) screen sizes.

Reply:
It works fine with the 1300px screen size. For very large screens we can consider using better image or SVG images in improvement phase. It's good to go for the first phase.